google/cloud/aiplatform/metadata/metadata_store.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from typing import Optional
+
+import logging
+from google.auth import credentials as auth_credentials
+from google.api_core import exceptions
+
+from google.cloud.aiplatform import base, initializer
+from google.cloud.aiplatform_v1beta1.types import metadata_store as gca_metadata_store
+from google.cloud.aiplatform_v1beta1.services.metadata_service import (
+    client as metadata_service_client,
+)
+
+
+class MetadataStore(base.AiPlatformResourceNounWithFutureManager):
```

Comment:
This should be a private class. We have the option to make it public in the future.